custom-tests.yaml

```diff
@@ -933,6 +942,23 @@ api:
   HTMLVideoElement:
     __base: var instance = document.createElement('video');
     __test: return bcd.testObjectName(instance, 'HTMLVideoElement');
+  IDBDatabase:
+    __base: >-
+      <%api.IDBOpenDBRequest:req%>
+      req.onsuccess = function() {
+        callback(req.result);
```

Comment:
In the context of these tests, `callback()` + `fail()` looks like an odd pairing. Why not `success()` or `result()` + `fail()` instead? Both success and failure uses callbacks, after all, so both could be called "callback" :)